changelog.json

```diff
@@ -1,5 +1,12 @@
 {
   "releases": {
+    "2.9.4-beta1": [
+      "[Added] Add a menu item to view a committed file change on GitHub - #12492. Thanks @haykam821!",
+      "[Added] Customizable high contrast theme - #12833",
+      "[Fixed] App no longer crashes on successful merge after conflicts resolved - #12888",
+      "[Improved] Use same width for hunk expansion handles as we do for line numbers - #12887",
```

Comment:
This sounds more like a fix to me, but in any case @niik tagged this with `no-notes`… since this is an user-facing fix, I'd say we add the release note anyway.